### DISCUSSION TITLE
A lot of German TV channels are missing. Could you please add them too?

### DISCUSSION THREAD

**Discussion by itsthomas** [iptv-org/community#425] [Q&A] [answered]
Hi,
Could you please add more German channels?
Such as:

Pro7
Kabeleins
RTL
RTL2
SAT1
NTV

**Comment by ztreamhd** [comment]
Except for NTV (which has a permanent, yet geo-blocked HLS link), none of these channels provide livestream sources for free or without registration, so there's not much we can do to add them.

**Reply by itsthomas** [reply]
Thanks for the info.
I used to watch IPTV german TV channels outside of Germany without any issues including the above channels. But about 5 years ago they added geo-blockades to stop people outside of Germany to watch german live TVs.
I still could use a VPN to bypass it and it was ok.
But it seems that at some point they have stopped streaming the above channels for free and you have to pay a subscription fee to watch them. Really sad.

**Comment by BellezaEmporium** [comment]
NTV is https://pntvlive-a.akamaized.net/live/ntvhome/dash/ntvhome.mpd but geo-locked.

**Reply by ztreamhd** [reply]
HLS stream for NTV: https://pntvlive-a.akamaized.net/live/ntvhome/hls/ntvhome.m3u8

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
